### PR TITLE
flintrock 0.9.0 (new formula)

### PR DESCRIPTION
**Pull request by tdsmith**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds [flintrock](https://github.com/nchammas/flintrock), a tool for launching Spark clusters on EC2.

A better test is unlikely because all other operations require AWS credentials.

### CODE REVIEW

File: Formula/flintrock.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    sha256 "70e8a77beed4562e7f14fe23a786b54f6296e34344c23bc42f07b15018ff98e9"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    ENV["LC_ALL"] = "en_US.UTF-8"
+    system bin/"flintrock"
```
Comment (fxcoudert) [inline Formula/flintrock.rb]: We'd need a test that does more. Can you try to perform an operation, and match the resulting failure? (error message and error code)

**Review by fxcoudert — DISMISSED** [review]

**Comment by tdsmith** [comment]
I added a test for a failure condition as requested.

(My two cents, fwiw: executing the main entry point of a Python CLI successfully is usually a pretty strong statement that the dependencies are in the right place; I don't think depending on the text of an error message really adds additional assurance.)

File: Formula/flintrock.rb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    msg, status = Open3.capture2e(bin/"flintrock", "destroy", "fascism")
+    assert_match /could not find your AWS credentials/, msg
+    assert_equal status.exitstatus, 1
```
Comment (ilovezfs) [inline Formula/flintrock.rb]:
```
    output = shell_output("#{bin}/flintrock destroy fascism 2>&1", 1)
    assert_match "could not find your AWS credentials", output
```

Comment (tdsmith) [reply Formula/flintrock.rb]: adjusted, thanks

**Review by ilovezfs — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @tdsmith

**Pull request closed without merging**